test/test_jit.py

```diff
@@ -11930,7 +11933,8 @@ def test_lstm_traced_cpu(self):
         inputs = get_lstm_inputs('cpu')
         try:
             ge = self.checkTrace(LSTMCellF, inputs)
-            self.assertExpectedGraph(ge.graph_for(*inputs))
+            graph = ge.graph_for(*inputs)
+            FileCheck().check("FusionGroup_0[device=-1]").run(str(graph))
```

Comment:
I think this is dead code. FusionGroup nodes do not have a device attribute anymore